interbotix_ros_slate/interbotix_slate_driver/src/slate_base_node.cpp

```diff
@@ -36,6 +36,7 @@ int main(int argc, char ** argv)
   auto node = std::make_shared<slate_base::SlateBase>();
 
   auto r = rclcpp::Rate(20);
+
```

Comment:
This update block could be turned into a (new) timer callback so that this node could be used as a composed node down the road.